README.md

```diff
@@ -8,33 +8,24 @@ A Jenkins plugin for automatically forwarding metrics, events, and service check
 
 ### Installation
 
-_This plugin requires [Jenkins 2.164.1][2] or newer._
+_This plugin requires [Jenkins 1.632][2] or newer._
 
 This plugin can be installed from the [Update Center][3] (found at `Manage Jenkins -> Manage Plugins`) in your Jenkins installation:
 
 1. Select the `Available` tab, search for `Datadog`, and select the checkbox next to `Datadog Plugin`.
 2. Install the plugin by using one of the two install buttons at the bottom of the screen.
 3. To verify the plugin is installed, search for `Datadog Plugin` on the `Installed` tab. 
-4. Create a [custom log source file][13]: create a `conf.yaml` inside `conf.d/jenkins.d` with the following:
-  ```
-  logs:
-
-    -type: tcp 
-     port: 10518 
-     service: <SERVICE>
-     source: jenkins
-  ```
-  
+
   Continue below for configuration.
 
 **Note**: If you see an unexpected version of the `Datadog Plugin`, run `Check Now` from the `Manage Jenkins -> Manage Plugins` screen.
 
 ### Configuration
 
-You can use two ways to configure your plugin to submit data to Datadog:
+There are two ways to configure your plugin to submit data to Datadog:
 
 * **RECOMMENDED**: Using a DogStatsD server / Datadog Agent that acts as a forwarder between Jenkins and Datadog.
-  - Build Logs collection only works with a full Datadog Agent installed.
+  - [Build Logs collection](#log-collection) only works with a full Datadog Agent installed.
```

Comment:
I don't think that's true, the http client forwards logs

Reply:
True, I will update with that!

Reply:
Updated!